core/shared/src/main/scala/fs2/text.scala

```diff
@@ -209,7 +209,7 @@ object text {
         }
 
         val isLast = r.isEmpty
-        (lastOutBuffer: Buffer).clear()
+        lastOutBuffer.clear()
```

Comment:
This is more about making a point :P we can revert to play it safe.

Reply:
Yeah let's revert these and the `flip` ones below. When we need to target 9+ I don't want to have to fix these all over again.